static-site/src/components/splash/userGroups.jsx

```diff
@@ -9,7 +9,7 @@ const UserGroups = (props) => {
 
   const colors = [...theme.titleColors];
 
-  const groupCards = props.user.groups.map((group) => {
+  const groupCards = props.user.visibleSources.map((group) => {
```

Comment:
how do we feel about calling the mapping variable `group` still?

Reply:
Just commented on that separately before I saw your note here. :-)